doc/Data Plane Code Generation/Subclients.md

```diff
@@ -0,0 +1,233 @@
+# Customizing the generated code
+
+## Subclients
+
+There are two categories of clients: service clients and their subclients. Service clients can be instantiated and have the Client suffix. Subclients can only be created by calling factory methods on other clients (commonly on service clients) and do not have the client suffix. The service client is the entry point to the API for an Azure service. Please refer to [guideline](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-subclients) for more detail information.
+
+There are two tools to organize subclients and service client.
```

Comment:
Do we provide tools to organize subclients and service client?

Reply:
@lirenhe  what tools do you meantion?

Reply:
I saw this sentense  'There are two tools to organize subclients and service client'